src/components/05_pages/Content/Content.js

```diff
@@ -34,6 +35,12 @@ const styles = {
     display: flex;
     flex-wrap: wrap;
   `,
+  addButton: css`
+    margin: 0.5rem;
+    position: fixed;
+    right: 0;
+    top: 0;
```

Comment:
Is there a specific reason for placing the add button to the top right corner? It seems like material UI is placing it in the bottom right corner in [their demo](https://material-ui.com/demos/buttons/#floating-action-buttons).

Reply:
Sorry, I just noticed you mentioned in the PR summary that you are not sure if it should be in the top right corner. 🙈In that case I probably would prefer to follow the material UI demo and move it to the bottom right corner. 🙂

Reply:
i found that bottom position of button not really obvious here. 
https://gyazo.com/fd38420bdc8c5ab346ecde6191a8d639

Reply:
maybe better to change 'edit' and 'delete' buttons variant? to have add button more 'special'
https://gyazo.com/66dd22b02ea1ed126d06e3e37ef8268c

Reply:
I think that is a good point. 👍 What do you think about using this pattern instead: https://material-ui.com/demos/buttons/#buttons-with-icons-and-label for the table rows?

Reply:
100% agree with @iberdinsky-skilld suggestion. The ones in the list shouldn't use the rounded style but the plain icon because otherwise they get too much attention and compete with the main action. Unless there's an accessibility issue with using icons without a label I'd use them alone.

Reply:
Definitely. Especially the red delete button dominates everything when using the rounded style.